mev_inspect/classifiers/specs/zero_ex.py

```diff
@@ -3,9 +3,17 @@
 )
 from mev_inspect.schemas.classifiers import (
     ClassifierSpec,
+    DecodedCallTrace,
+    SwapClassifier,
 )
 
 
+class ZeroXSwapClassifier(SwapClassifier):
+    @staticmethod
+    def get_swap_recipient(trace: DecodedCallTrace) -> str:
+        return trace.inputs["recipient"]
```

Comment:
there's aways a recipient specified or does it ever default to from address?

Reply:
this was actually incorrect, but i've corrected it